lib/models/tts.dart

```diff
@@ -73,6 +74,20 @@ class TtsModel extends ChangeNotifier {
     notifyListeners();
   }
 
+  void addNameToBlacklist(String name) {
+    blacklist.add(name);
+    ttsHandler.blackList = blacklist;
```

Comment:
I don't think this assignment is necessary

Reply:
`ttsHandler` will not have the updated set if I don't use assignment to update the set.

Reply:
I see. I think we could make this a passthrough call then, otherwise we're maintaining the muted users set in two places. ie do

```
ttsHandler.mutedUsers.add(user);
notifyListeners();
```

and don't maintain the set of muted users in the model. It's not actually strictly necessary.